core/src/main/java/kafka/server/share/SharePartition.java

```diff
@@ -1716,7 +1725,17 @@ private KafkaException fetchPersisterError(short errorCode, String errorMessage)
         }
     }
 
-    private AcquisitionLockTimerTask scheduleAcquisitionLockTimeout(String memberId, long firstOffset, long lastOffset) {
+    //Visible for Testing
+    public AcquisitionLockTimerTask scheduleAcquisitionLockTimeout(String memberId, long firstOffset, long lastOffset) {
```

Comment:
Can it be non-public but default if has to be visible in SharePartition tests?

```suggestion
    AcquisitionLockTimerTask scheduleAcquisitionLockTimeout(String memberId, long firstOffset, long lastOffset) {
```